spec/models/like.rb

```diff
@@ -0,0 +1,12 @@
+require 'rails_helper'
```

Comment:
- [ ] **Required:** Just as the previous reviewers have stated, kindly ensure that all files in the spec folder, i.e test files end with spec. For example `like_spec.rb`

Reply:
Hi @brytebee,  I don't have any chance to submit it my project,
I have submitted tree time, i think it's not big problem to reject my project
how can submit it again ?